unicodetools/src/main/resources/org/unicode/props/ExtraPropertyValueAliases.txt

```diff
@@ -85,6 +85,10 @@
 # @missing: 0000..10FFFF; Emoji_Component  	; No
 # @missing: 0000..10FFFF; Extended_Pictographic  	; No
 
+# @missing: 0000..10FFFF; kEH_Core  ;  No
+# @missing: 0000..10FFFF; kEH_NoMirror  ;  No
+# @missing: 0000..10FFFF; kEH_NoRotate  ;  No
```

Comment:
Do we need `@missing` lines for binary properties? They must default to "No" because we don't currently have syntax to have a "No" override.

Reply:
For now we need them, though this annoys me:
https://github.com/unicode-org/unicodetools/blob/5aa08acba04b6e5794ead18e03c355ae1417670a/unicodetools/src/main/resources/org/unicode/props/ExtraPropertyValueAliases.txt#L6